app/assets/stylesheets/user_new.scss

```diff
@@ -0,0 +1,26 @@
+@import "colors";
+
+#new_user {
```

Comment:
Avoid using id selectors<br>Selector `new_user` should be written in lowercase with hyphens